File: Segitiga (Fill)/C/segitiga-fill-i.c
#include <stdio.h>
int main(){
    int x;
    printf("Masukan Tinggi Segitiga : ");
    scanf("%x", &x);
    for(int i = 1; i <= x; i++){
        for(int j = x; j >= i; j--){
            printf(" ");
        }
        
        for(int l = 2; l <= i; l++){
            printf("*");
        }
        
        for(int k = 1; k <= i; k++){
                printf("*");
            }
        printf("\n");
    }
    return 0;
}

File: Segitiga (Fill)/GO/segitiga-fill-i.go
package main
import "fmt"

func main(){
    var x int
    fmt.Print("Masukan Tinggi Segitiga: ")
    fmt.Scan(&x)
    
    for i := 1; i <= x; i++ {
        for j := x; j >= i; j-- {
            fmt.Printf(" ")
        }
        
        for l := 2; l <= i; l++ {
            fmt.Printf("*")
        }
        
        for k := 1; k <= i; k++ {
            fmt.Printf("*")
        }
        fmt.Printf("\n")
    }
}
